modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="configuring_remote_write_storage_{context}"]
+= Configuring remote write storage
+
+You can configure remote write to enable Prometheus to send ingested metrics to remote systems for long-term storage. 
+Doing so has no impact on how or for how long Prometheus stores metrics using persistent local storage.
```

Comment:
`persistent local storage`
change to
`persistent storage` is better

Reply:
Updated. Thanks!